File: 2278-find-three-consecutive-integers-that-sum-to-a-given-number/2278-find-three-consecutive-integers-that-sum-to-a-given-number.cpp
class Solution {
public:
    vector<long long> sumOfThree(long long num) {
        vector<long long>n;
        if(num%3!=0){
            return n;
        }
        n.push_back(floor(num/3)-1);
        n.push_back(floor(num/3));
        n.push_back(floor(num/3)+1);
        return n;

        
    }
};

File: 2278-find-three-consecutive-integers-that-sum-to-a-given-number/2278-find-three-consecutive-integers-that-sum-to-a-given-number.py
class Solution:
    def sumOfThree(self, num: int) -> List[int]:
        k=0
        n=num
        l=[]
        while(num):
            rem=num%10
            k+=rem
            num//=10
        if(k%3==0):
            l.append((n//3)-1)
            l.append(n//3)
            l.append((n//3)+1)
        return l
            
        
        
        